src/status_im/subs/wallet/wallet.cljs

```diff
@@ -527,6 +527,12 @@
  (fn [ui]
    (get-in ui [:account-page :active-tab])))
 
+(rf/reg-sub
+ :wallet/home-tab
+ :<- [:wallet/ui]
+ (fn [ui]
+   (:active-tab ui)))
```

Comment:
This is what I was referring to in the previous comment, it could be:

`(or (:active-tab ui) first-tab-id)`

Reply:
what about putting it in the initial db state?